docs/introduction/Car-registry-tutorial.rst

```diff
@@ -2,73 +2,76 @@
 |                     |                                                                                                                                                                                                                       | proof -- proof for open initial car box                                                                                                                                                                             |
 |                     |                                                                                                                                                                                                                       | price -- selling price                                                                                                                                                                                              |
 |                     |                                                                                                                                                                                                                       | buyerProposition -- current implementation expect to have the specific buyer which had been found off chain. Thus during creation of car sell order we already know buyer and shall put his future car proposition  |
 +---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| CarBuyOrderInfo     | Data required for buying a car or recall a car sell order. Used in transaction processing.                                                                                                                            | carSellOrderBoxToOpen -- Car sell order box to be open                                                                                                                                                              |
+| CarBuyOrderInfo     | Data required for buying a car or recalling a car sell order. Used in transaction processing.                                                                                                                         | carSellOrderBoxToOpen -- Car sell order box to be open                                                                                                                                                              |
```

Comment:
| CarBuyOrderInfo     | Is data required for buying a car or recalling a CarSellOrder. Used in transaction processing.                                                                                                                         | carSellOrderBoxToOpen -- A CarSellOrder box to be opened.                                                                                                                                                              |